### DISCUSSION TITLE
Running opendatacam on MacOS

### DISCUSSION THREAD

**Discussion by mkhammar** [opendatacam/opendatacam#586] [Q&A]
Hi,

Is there a workaround to run it from MacOS ?

Thanks

**Comment by vsaw** [comment]
For Intel Macs, you could use the [cpu-amd64](https://hub.docker.com/layers/opendatacam/opendatacam/development-cpu-amd64/images/sha256-ef396cc2089de95afa4250d48d18b8e8e40203093516a68c9d4a9feaab12f964?context=explore) docker image, which will run ODC on your CPU. Depending on machine this may give you 3-5 fps on Yolov4-tiny.

I don't have a solution for Apple Silicon yet. (Maybe the same docker-image also works on Apple Silicon, I can't test as I don't have a M1 Mac)

Background:
OpenDataCam uses Darknet as it's neural network. Darknet only supports Intel CPU or NVIDIA GPUs. But NVIDIA stopped supporting Apple in 2019 (as apple stopped offering NVIDIA graphics in their devices). Hence the impasse.
